Lib/http/client.py

```diff
@@ -593,8 +593,8 @@ def _read_chunked(self, amt=None):
                     amt -= chunk_left
                 self.chunk_left = 0
             return b''.join(value)
-        except IncompleteRead:
-            raise IncompleteRead(b''.join(value))
+        except IncompleteRead as incomplete_read:
+            raise IncompleteRead(b''.join(value)) from incomplete_read
```

Comment:
```suggestion
        except IncompleteRead as exc:
            raise IncompleteRead(b''.join(value)) from exc
```
follow (informal) code conventions used by stdlib